src/screen/ConnectionsList/Connection/styles.ts

```diff
@@ -52,6 +56,53 @@ export const Container = styled.li<IContainerProps>`
       `}
   }
 
+  .connection-actions-menu {
+    width: 220;
+    z-index: 1;
+    padding: 8px 0;
+    background: ${props => props.theme.backgrounds.darker};
+    border: 1px solid ${props => props.theme.backgrounds.lightest};
+    border-radius: 5px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+
+    > div {
+      display: flex;
+      align-items: center;
+      padding: 8px 16px;
+      cursor: pointer;
+
+      :hover {
+        background: ${props => props.theme.backgrounds.dark};
+      }
+
+      svg {
+        margin-right: 8px;
+        color: #9696a7;
+      }
+
+      :first-child {
```

Comment:
Maybe we should create a ConnectButton and DisconnectButton with their own styles instead of changing it's styles conditionally using the :first-child, what do you think?

Reply:
Yes, that'd be better.